### ISSUE TITLE
Path to Haxe executable is not valid

### ISSUE THREAD

**Issue by nsmarkop** [vshaxe/vshaxe#391]
I just installed Haxe 4.0.1 for Windows from the website:

https://haxe.org/download/

And then installed the Haxe plugin for VSCode but I keep getting the following error message: `Path to Haxe executable is not valid: 'haxe.exe'. Please check your settings.`

I've verified haxe.exe is in my PATH by using it in the VSCode terminal to compile the plugin's generated project template. I updated the path in my settings to point to the executable directly instead at `C:\\HaxeToolkit\\haxe\\haxe.exe` and received the same error message still.

Looking at the Haxe output window in VSCode only shows me that the language server started and fails due to code -32601 for everything. Right clicking in VSCode -> Run Code seems to work though with the expected output:

```
[Running] haxe --cwd "e:\projects\haxe\helloworld\src" --run Main
Main.hx:3: Hello, world!

[Done] exited with code=0 in 0.193 seconds
```

**Comment by nsmarkop** [comment]
Looks like I figured out how to make it work, though I'm unsure why.

I had a workspace open with several projects' (non-Haxe) folders in it, one being the Haxe one I was using for testing out the project template. Once I closed the workspace and just opened just the folder instead the extension started working as expected.

I didn't have any workspace-specific configuration for Haxe as far as I could tell.
